config/environments/development.rb

```diff
@@ -97,3 +97,9 @@
   end
 
 end
+
+
+ActiveMerchant::Billing::Base.mode = :test
+
+  ::GATEWAY = ActiveMerchant::Billing::StripeGateway.new(
+    :login => ENV['SECRET_KEY'])
```

Comment:
Please have a look at how [OSEM is configued](https://github.com/openSUSE/osem/blob/master/INSTALL.md#configure) with environment variables and `config/secrets.yml.example`